### DISCUSSION TITLE
Exporting High-Quality mesh from NeRF scans

### DISCUSSION THREAD

**Discussion by smtabatabaie** [NVlabs/instant-ngp#1358] [General]
Hi everyone, I just trained my first NeRF and it looks good and everything's fine and great. The problem I have right now is that I want to use these scans in my AR/VR projects and in Unity, and the mesh export I get from instant-ngp is very low quality and not photo-realistic at all. I wanted to ask if you have any tips to export a photo-realistic 3d model from my NeRF scans, whether from instant-ngp or any other methos. Would really appreciate if someone can help me with this.
Thanks very much

**Comment by JoeLoginIsAlreadyTaken** [comment]
Yes it is possible, but it requires quite a lot post processing.
The quality of the Nerf is also very important, and this depends mostly on the quality of your image source.
I have described my process on exporting and first mesh cleanup here: https://github.com/NVlabs/instant-ngp/issues/1333
In Blender I clean up the Mesh (remesh, sculpting, decimate). After that I create a new texture and paint it using the original images as stencil. Here is a good video on this: https://www.youtube.com/watch?v=PansYUHh-yo
This is not exactly like the original, but for me this work better than converting the Vertex Color into a Texture.

![0199](https://github.com/NVlabs/instant-ngp/assets/35426186/3a8d1de9-c1cb-47d0-a9e6-a5b2cfdd1e4d)
![loewe_blender](https://github.com/NVlabs/instant-ngp/assets/35426186/ca135d3f-cbda-4f21-9c03-3e8193489944)
